### PR TITLE
Add Jenkinspipline for testing

### PR DESCRIPTION
**Pull request by Scriptkiddi**
Signed-off-by: Fritz Otlinghaus <fritz@otlinghaus.it>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,10 +1,145 @@
...
+            }
+          githubNotify context: 'Building PR', description: 'Building pull request',  status: 'SUCCESS'
+        }
+        post{
+          failure{
+            githubNotify context: 'Building PR', description: 'Building pull request',  status: 'FAILURE'
+          }
+        }
+      }
+      stage('Run Bosch checks') {
```
Comment (aedelmann) [inline Jenkinsfile]: Change to Run Compliance Checks

File: Jenkinsfile [hunk 2]
```diff
@@ -1,10 +1,145 @@
...
+              }
+            }
+          }
+          stage("CLMScan"){
+            steps{
+              githubNotify context: 'CLMScan', description: 'Running CLMScan',  status: 'PENDING'
+                withMaven(
+                    maven: 'maven-latest',
+                    mavenLocalRepo: '.repository') {
+                  catchError {
```
Comment (aedelmann) [inline Jenkinsfile]: Add Todo:

File: Jenkinsfile [hunk 3]
```diff
@@ -1,10 +1,145 @@
...
+                    withCredentials([usernamePassword(credentialsId: 'CLMScanUser', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
+                      nexusPolicyEvaluation failBuildOnNetworkError: false, iqApplication: selectedApplication('vorto-repository'), iqStage: 'build', jobCredentialsId: 'CLMScanUser'
+                        //      s3Upload(file:'file.txt', bucket:'pr-vorto-documents', path:'repository/repository-server/target/**/*.pdf')
```
Comment (aedelmann) [inline Jenkinsfile]: Add Todo for report

File: Jenkinsfile [hunk 4]
```diff
@@ -1,10 +1,145 @@
...
+              }
+            }
+          }
+          stage("AVScan infomodelrepository"){
+            steps{
+              githubNotify context: 'AVScan Infomodel', description: 'Running AntiVirus Scan on infomodelrepository.jar',  status: 'PENDING', targetUrl: ""
+                // Get Bosch pom files to run in an extra folder to keep the open source project clean and because the Bosch maven plugins can not be licensed under EPL
+                dir('avscan_infomodel') {
+                  //copy files over to the new maven folder to run AntiVirus Scans
+                  git url: "https://github.com/Scriptkiddi/vorto_bosch_jenkins.git"
```
Comment (aedelmann) [inline Jenkinsfile]: Change repo

File: Jenkinsfile [hunk 5]
```diff
@@ -1,10 +1,145 @@
...
+                    sh 'cp ../repository/repository-server/target/infomodelrepository.jar ./'
+                }
+              withMaven(
+                  maven: 'maven-latest',
+                  mavenLocalRepo: '.repository') {
+                sh 'mvn verify -Dbosch.avscan.fileToScan=infomodelrepository.jar -f avscan_infomodel/pom_bosch.xml'
+                  withAWS(region:'eu-central-1',credentials:'aws-s3-vorto-jenkins-technical-user') {
+                    s3Upload(file:'avscan_infomodel/target/inl-releng-avsupport/avscan_report.html', bucket:'pr-vorto-documents', path:"avscans/${CHANGE_ID}/${BUILD_NUMBER}/infomodelrepository_report.html")
```
Comment (aedelmann) [inline Jenkinsfile]: Remove Bosch servers from report

File: Jenkinsfile [hunk 6]
```diff
@@ -1,10 +1,145 @@
...
+            }
+          }
+          stage("AVScan generator-runner"){
+            steps{
+              githubNotify context: 'AVScan Generators', description: 'Running AntiVirus Scan on generator-runner-exec.jar',  status: 'PENDING', targetUrl: ""
+                // Get Bosch pom files to run in an extra folder to keep the open source project clean and because the Bosch maven plugins can not be licensed under EPL
+                dir('avscan_generator') {
+                  //copy files over to the new maven folder to run AntiVirus Scans
+                  git url: "https://github.com/Scriptkiddi/vorto_bosch_jenkins.git"
+                    sh 'cp ../generators/generator-runner/target/generator-runner-exec.jar ./'
```
Comment (aedelmann) [inline Jenkinsfile]: Remove bosch url from report and change repo

**Comment by Scriptkiddi** [comment]
all done

**Pull request merged**